server/server.js

```diff
@@ -1,148 +1,17 @@
-// Data feed imports
-const { sectorInformation } = require("./components/sectorInformation");
-const { topPeers } = require("./components/topPeers");
-const { companyOverview } = require("./components/companyOverview");
-const { latestNewsInterval } = require("./components/latestNewsInterval");
-const { keyStats } = require("./components/keyStats");
-const { stockTicker } = require("./components/stockTicker");
-const { chartData } = require("./components/chartData");
-
+const path = require("path");
+require("dotenv").config({ path: path.resolve(__dirname, ".env") });
```

Comment:
What does the `path` library do, and why do we need it here?

Reply:
It used to work with files and directory path. Because previously, `.env` is in server folder which cannot read from `server/server`, if we resolve path, it can read from `server/server`

Reply:
Tighter on the terminology, please.
`dotenv` looks for a `.env` file in the current working directory, which is wherever the JS file was executed from.
When running `node ./server/server`, it cannot find the `.env` file because it is nested inside the `server` folder and not in the current working directory.